### PR TITLE
Niu keymap and README update

### PR DESCRIPTION
**Pull request by yttyx**
## Description

Added steno layer. Finalised other layers following months of use. Tidied README file.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/niu_mini/keymaps/yttyx/keymap.c [hunk 1]
```diff
@@ -1,141 +1,104 @@
...
+}
+
+
+
+
+
+
+
+
+
```
Comment (fauxpark) [inline keyboards/niu_mini/keymaps/yttyx/keymap.c:104]:
```suggestion
```

File: keyboards/niu_mini/keymaps/yttyx/keymap.c [hunk 2]
```diff
@@ -1,141 +1,104 @@
 
 #include QMK_KEYBOARD_H
+#include <keymap_steno.h>
```
Comment (fauxpark) [inline keyboards/niu_mini/keymaps/yttyx/keymap.c:3]:
```suggestion
#include "keymap_steno.h"
```

Comment (yttyx) [reply keyboards/niu_mini/keymaps/yttyx/keymap.c]: Fixed

File: keyboards/niu_mini/keymaps/yttyx/keymap.c (hunk 1 above)
Comment (yttyx) [reply keyboards/niu_mini/keymaps/yttyx/keymap.c]: Fixed

**Comment by yttyx** [comment]
Hi drashna and fauxpark - thank you for the review. Changes made. Regards, Nick.

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**